### DISCUSSION TITLE
Pnp Search thtowing error: [HTTP]:500 - [CorrelationId]:ba4caca0-10b4-6000-6637-908b5d6eb7d0 [Version]:16.0.0.23605

### DISCUSSION THREAD

**Discussion by AndyOppelaar** [microsoft-search/pnp-modern-search#2926] [Q&A]
Since today I am getting this error on PnP Search version 4.8.0.0 on every site when adding a Search results box
[HTTP]:500 - [CorrelationId]:ba4caca0-10b4-6000-6637-908b5d6eb7d0 [Version]:16.0.0.23605

![image](https://user-images.githubusercontent.com/131661619/233959782-6347ee55-da89-4b28-aa78-a3bd86ac99c4.png)

**Comment by kasperbolarsen** [comment]
general outage
![image](https://user-images.githubusercontent.com/20593570/233974807-9b2705fd-0d30-45ef-bba2-9e9f77e30d98.png)

**Comment by AndyOppelaar** [comment]
Ah, thanks. I'll wait :-)

**Comment by AtibardAurelien** [comment]
I had the same issue today. @kasperbolarsen : Do you have any recent information on this incident?

**Comment by kasperbolarsen** [comment]
The last update seems to be from 01pm UTC. If you have access to a tenant then see https://admin.microsoft.com/Adminportal/Home?source=applauncher#/servicehealth/:/alerts/MO545600

**Comment by kasperbolarsen** [comment]
![image](https://user-images.githubusercontent.com/20593570/234196130-af742bb9-ee1f-4b4d-9fa3-e5caa04ec4c3.png)

**Comment by afnepipe** [comment]
I'm getting the same issue today May 14th.

**Reply by afnepipe** [reply]
@kasperbolarsen @wobba 
Guys, I hope you are doing well.
Can you give me some information about this issue?  One customer is receiving this kind of message continuously.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/96144314/c22b7cb5-2870-4f6d-849e-01cbf2a9efbe)


Thanks for your answer.

**Reply by kasperbolarsen** [reply]
The reason for that issue was an error at the API level. So the error had nothing to do with PnP Modern Search,  it just became visible here first.
Have you checked if there is anything in the service messages in the tenant.

**Reply by afnepipe** [reply]
Thanks @kasperbolarsen to answer me.
What kind of service messages?
Unfortunately I'm not SharePoint admin, so I cannot see those services messages?

**Reply by kasperbolarsen** [reply]
In that case you will have to ask one of your admins.
I also follow @MSFT365Status on X, they are excellent
